adminSiteClient/VariablesAnnotationTypesAndUtils.tsx

```diff
@@ -183,3 +273,244 @@ export const getItemStyle = (
+        else return operation
+    } else if (filterTree.type === "rule") {
+        return (
+            match(filterTree.properties.operator)
+                // If we have a rule, check what operator is used and build the corresponding operation
+                .when(
+                    (op) => op && getComparisonOperator(op),
+                    (op) => {
+                        const operator = getComparisonOperator(op as string)
+                        const field = getFieldSymbol(
+                            filterTree.properties.field!
+                        )
+                        if (
+                            filterTree.properties.value.length === 0 ||
+                            filterTree.properties.value[0] === undefined
+                        )
+                            return undefined
+                        const val = getValueAtom(filterTree.properties.value[0])
+                        return new NumericComparision(operator!, [field, val!])
+                    }
+                )
+                .when(
+                    (op) => op && getEqualityOperator(op),
+                    (op) => {
+                        const operator = getEqualityOperator(op as string)
+                        const field = getFieldSymbol(
+                            filterTree.properties.field!
+                        )
+                        const val = getValueAtom(filterTree.properties.value[0])
+                        if (val === undefined) return undefined
+                        return new EqualityComparision(operator!, [field, val])
+                    }
+                )
+                .when(
+                    (op) => op && op === "like",
+                    () => {
+                        const field = getFieldSymbol(
+                            filterTree.properties.field!
+                        )
+                        if (
+                            filterTree.properties.value.length === 0 ||
+                            filterTree.properties.value[0] === undefined
+                        )
+                            return undefined
+                        const val = new StringAtom(
+                            filterTree.properties.value[0]
+                        )
+                        return new StringContainsOperation(field, val)
+                    }
+                )
+                .when(
+                    (op) => op && getNullCheckOperator(op as string),
+                    (op) => {
+                        const operator = getNullCheckOperator(op as string)!
+                        const field = getFieldSymbol(
+                            filterTree.properties.field!
+                        )
+                        return new NullCheckOperation(operator!, field)
+                    }
+                )
+                .with("is_empty", "is_not_empty", (operator) => {
+                    const field = getFieldSymbol(filterTree.properties.field!)
```

Comment:
Can pull the `getFieldSymbol` out before the `match`, since it's used in all cases